Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "rails", "6.1.5"
+gem "rails", "~> 7.0.2"
```

Comment:
We should fix this at a specific release, otherwise we run a risk of rails being downgraded if a dependency becomes incompatible.

Reply:
Yep — had done this before I saw the discussion on Slack. Will update.